docs/appendices/release-notes/unreleased.rst

```diff
@@ -43,7 +43,9 @@ Unreleased Changes
 Breaking Changes
 ================
 
-None
+- Added duplicate column check to
+  :ref:`named index column definition <named-index-column>`. Duplicate entry
+  used to be silently ignored, now it shows an error message.
```

Comment:
```suggestion
- Raise an exception if duplicate columns are detected on
  :ref:`named index column definition <named-index-column>` instead of
  silently ignoring them.
```